src/shared/components/workbasket-filters/workbasket-filters.component.spec.ts

```diff
@@ -618,6 +618,35 @@ describe('WorkbasketFiltersComponent', () => {
       expect(arg['formGroup'].value).toEqual(formGroup.value);
       expect(workbasketHandler.applyFilters).toHaveBeenCalledTimes(1);
     }));
+
+    it ('should update form group filters', () => {
+      const formGroupLocalStorage = {
+        regionList: 'london',
+        londonFRCList: 'london',
+        londonCourtList: 'FR_londonList_10',
+        southEastFRCList: null,
+        thamesvalleyCourtList: null
+      };
+      windowService.getLocalStorage.and.returnValue(JSON.stringify(formGroupLocalStorage));
+
+      const formControls = {
+        regionList: new FormControl('southeast'),
+        londonFRCList: new FormControl('london'),
+        londonCourtList: new FormControl('FR_londonList_10'),
+        southEastFRCList: new FormControl('thamesvalley'),
+        thamesvalleyCourtList: new FormControl('FR_thamesvalleyList_2')
+      };
+      component.formGroup = new FormGroup(formControls);
+
+      component.updateFormGroupFilters();
+      console.log(component.formGroup.get('regionList').value);
+      console.log(component.formGroup.get('southEastFRCList').value);
+      console.log(component.formGroup.get('thamesvalleyCourtList').value);
+      console.log(component.formGroup.get('londonFRCList').value);
+      console.log(component.formGroup.get('londonCourtList').value);
```

Comment:
remove all the console logging please

Reply:
I have removed all the console.log statements. Thank you.